### ISSUE TITLE
Have a question about how do I get the data set

### ISSUE THREAD

**Issue by REMEATGreens** [copenlu/weak-labeler#1]
I am using git lfs for the first time and there is an error when using git lfs: “This repository is over its data quota. Account responsible for LFS bandwidth should purchase more data packs to restore access. "I don't know how to fix it.

Maybe I can use the data in the original paper, but I don't know how you processed the data in detail...

**Comment by osoblanco** [comment]
We managed to currently host the datasets proposed and processed by [Hardalov et al.](https://arxiv.org/abs/2104.07467) seperately [here](https://drive.google.com/drive/folders/1jxqgiCGwzNkkRa9THN7vZH-nzJwjHVx1?usp=drive_link).

We use the processed data as is without further modifications.

**Comment by REMEATGreens** [comment]
> We managed to currently host the datasets proposed and processed by [Hardalov et al.](https://arxiv.org/abs/2104.07467) seperately [here](https://drive.google.com/drive/folders/1jxqgiCGwzNkkRa9THN7vZH-nzJwjHVx1?usp=drive_link).
> 
> We use the processed data as is without further modifications.

Thank you so much,but I noticed that the Snopes folder in the Google Drive share appears to be empty. I was wondering if this was intentional or if there might be an issue with the sharing.

**Comment by osoblanco** [comment]
Hi @shengdan320 

There seems to be an issue with where the dataset was stored. Trying to recover it.

**Comment by REMEATGreens** [comment]
Thanks, and I also found some questions in the README：

----- Google drive seems not share all the data,  for example：
```
'python weaklabeler/fewShot/train.py --experiment_name fewShot_training_topic_new --feat_extractor roberta-base 
--training_path weaklabeler/Data/Golden_set/few_shot_training_data_renamed.csv  --valid_path weaklabeler/Data/psudo_annotated_few.csv ...'
```
-training_path weaklabeler/Data/Golden_set"can not find the file named few_shot_training_data_renamed.csv, 
 and valid_path  psudo_annotated_few.csv seems also...

in ood.sh , `--training_path weaklabeler/Data/stance_data/stance_data.csv --model_save_path weaklabeler/models/ --valid_path weaklabeler/Data/stance_data/stance_test_data_sample.csv` do these different Datasets use the same 'Data/stance_data/stance_data.csv' file for each 'echo' training?








----- Details about model training:

1. What I understand In the paper is that in-domain(few shot?) Train is 16 kinds of data for training and then testing. In the Out-of-domain training (zero shot?)process, is one kind of data selected as the test set and the other 15 kinds as the training set? Is that right?

2.  In README "An example use of the few-shot pipeline can look like this" section, what is it used for, Is 'covid_currated.csv' a dataset containing multi domain as I understand above.Or are they just Covid related datasets, but do the same operations as the 16 datasets above? Is there any code in the warehouse to process data and sampling data?    

3. Section 'An example use of the few-shot pipeline can look like this' and Section 'To train the model on the few-shot dataset, use the following command'.,Is there any difference in the use of the two parts

4.  Otherwise,How are the two pth files"fewShot_currated_stance.pth""fewShot_psudoannot_topic.pth" in the models folder obtained，what are they used for.What’s the meaning of their names.

       


I'm sorry to trouble you but  I am very interested in your work,could you please solve my problems?

**Comment by osoblanco** [comment]
Hi @shengdan320 

As the project was part of a much longer effort, the README was not updated for a short while, causing this confusion. I went ahead and updated all of the commands. Hope this resolves the issue.

**Comment by REMEATGreens** [comment]
really appreciate it
